### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by martingagnon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Trikot.patron
+
+This repository is the stable base upon which we build our Elixir projects at Mirego.
+We want to share it with the world so you can build awesome Elixir applications too.
```
Comment (remi) [inline README.md:4]: Elixir? 😜

Comment (marcantoinefortier) [reply README.md:4]: Are you ok Martin? 😅

Comment (antoinelamy) [reply README.md]: Fixed

**Review by alarochelle — APPROVED** [review]

**Review by gbourassa — APPROVED** [review]

**Comment by martingagnon** [comment]
👍

**Pull request merged**